app/promptfoo/promptfooconfig.ci.yaml

```diff
@@ -26,6 +26,7 @@ defaultTest:
 evaluateOptions:
   delay: 1000
   showProgressBar: true
+  maxConcurrency: 1
```

Comment:
@fg-nava Made the suggested change but still [seeing more than 1 thread](https://github.com/navapbc/labs-decision-support-tool/actions/runs/15425101003/job/43409924283#step:10:57)

Reply:
Feel free to push changes to this PR.